src/test/java/dev/aziz/movies/services/MovieServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package dev.aziz.movies.services;
+
+import dev.aziz.movies.dtos.MovieDto;
+import dev.aziz.movies.dtos.ReducedMovieDto;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class MovieServiceTest {
+
+    @Mock
+    private MovieService movieService;
+
+    @Test
+    void getMoviesTest() {
+        // given
+        List<ReducedMovieDto> reducedMovieDtoList = List.of(
+                new ReducedMovieDto("Titanic", 1997, 5),
+                new ReducedMovieDto("The Shawshank Redemption", 1994, 4),
+                new ReducedMovieDto("The Godfather",1972, 3)
+        );
+        Mockito.when(movieService.getMovies()).thenReturn(reducedMovieDtoList);
```

Comment:
You can't mock the method you want to test.

Reply:
So 31 and 47 lines are not needed because there is no repo yet? Do I understand right?